### DISCUSSION TITLE
issue with template_synology_diskstation_snmpv3

### DISCUSSION THREAD

**Discussion by sebcapside** [zabbix/community-templates#156] [General]
Hi all, I got some issue with 
template_synology_diskstation_snmpv3
Zabbix 6.2
snmpv3 ok.
import yaml file is on error with missing ICMP Ping
event if I force this erro
I got another trigger expression incorrect...

**Comment by t984447** [comment]
I had this issue also, where I found the solution to edit the file and rename it to the defaulkt  installed template  named "ICMP Ping"

was:

`      templates:
        -
          name: 'Template Module ICMP Ping'`



`      templates:
        -
          name: 'ICMP Ping'`
